### PR TITLE
 Fix logger for stdout attribute

### PR DESCRIPTION
**Pull request by hill-a**
* fix for an issue raised in #242

### CODE REVIEW

File: stable_baselines/logger.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def __init__(self, filename_or_file):
             self.file = open(filename_or_file, 'wt')
             self.own_file = True
         else:
-            assert hasattr(filename_or_file, 'read'), 'expected file or str, got %s' % filename_or_file
+            assert hasattr(filename_or_file, 'write'), 'expected file or str, got %s' % filename_or_file
```
Comment (araffin) [inline stable_baselines/logger.py]:
```suggestion
            assert hasattr(filename_or_file, 'write'), 'Expected file or str, got {}'.format(filename_or_file)
```

**Comment by araffin** [comment]
@hill-a is the feature already tested? (I will check coday coverage report)

File: docs/misc/changelog.rst [hunk 2]
```diff
@@ -9,6 +9,7 @@ Pre-Release 2.5.1a0 (WIP)
 --------------------------
 
 - doc update (fix example of result plotter + improve doc)
+- fixed plotter issues when stdout lacks `read` function
```
Comment (araffin) [inline docs/misc/changelog.rst]:
```suggestion
- fixed plotter issues when stdout lacks ``read`` function
```

double back ticks for `.rst` files ...

Comment (hill-a) [reply docs/misc/changelog.rst]: still not used to that.. Thanks

File: docs/misc/changelog.rst [hunk 3]
```diff
@@ -9,6 +9,7 @@ Pre-Release 2.5.1a0 (WIP)
 --------------------------
 
 - doc update (fix example of result plotter + improve doc)
+- fixed plotter issues when stdout lacks ``read`` function
```
Comment (araffin) [inline docs/misc/changelog.rst]:
```suggestion
- fixed logger issues when stdout lacks ``read`` function
```

this is the logger right?

**Pull request merged**